src/Transports.AspNetCore/WebSockets/IWebSocketAuthenticationService.cs

```diff
@@ -0,0 +1,25 @@
+using GraphQL.Transport;
+using Microsoft.AspNetCore.Http;
+
+namespace GraphQL.Server.Transports.AspNetCore.WebSockets;
+
+/// <summary>
+/// Authenticates an incoming GraphQL over WebSockets request with the
+/// connection initialization message.  A typical implementation will
```

Comment:
What is _ connection initialization message_ ?

Reply:
The ConnectionInit message.  Again, the only purpose of the ConnectionInit message is to allow authentication/authorization for the connection, since browsers cannot send the HTTP 'Authorization' header to a WebSocket endpoint.